tools/developer/taptest.sh

```diff
@@ -52,14 +52,10 @@ echo "VERSION ${VERSION}"
 QUIET="-v"
 QUIET="-q"
 
-PGPORT="${PGPORT:-5432}"
-PGUSER="${PGUSER:-$USER}"
-
```

Comment:
Actually this was done so that the user can adjust the port on their own copy of run.sh when there is a change of port or user.

Undo this change